src/core/linter-rules/check-internal-slots.js

```diff
@@ -23,10 +23,12 @@ const lang = defaultLang in meta ? defaultLang : "en";
  * @param {Document} doc The document to be checked.
  */
 function linterFunction(conf, doc) {
-  const offendingElements = [...doc.querySelectorAll("var+a")].filter(
-    checkElement
+  const filteredElements = [...doc.querySelectorAll("var+a")].filter(
```

Comment:
So, this is still a bit odd... "filterElement" is checking that the `<a>`matches "[[\w]]", and then `checkElement` is checking for "starts with .[[\w]]" (which is somewhat redundant, given the first check). That makes me think the two could still be combined (and filterElement should go away)? Or am I missing something?

Reply:
Ah, apologies for the miscommunication. For now, we should only treat `<var>foo</var><a>[[foo]]</a>` (i.e., missing "." between `<a>` and `<var>`) as erroneous.